integration_tests/e2e/deleteContactEmail.cy.ts

```diff
@@ -53,7 +53,7 @@ context('Delete Contact Email', () => {
     Page.verifyOnPage(EditContactMethodsPage, 'First Middle Names Last') //
       .clickDeleteEmailLink('first@example.com')
 
-    Page.verifyOnPage(ConfirmDeleteEmailPage) //
+    Page.verifyOnPage(ConfirmDeleteEmailPage, 'First Middle Names Last') //
       .hasEmailAddress('first@example.com')
```

Comment:
What are the // characters on the ends of the lines? Just empty comments, or serving another purpose?